Kernel/FileSystem/VirtualFileSystem.h

```diff
@@ -35,6 +35,8 @@ struct UidAndGid {
 class VirtualFileSystem {
     AK_MAKE_ETERNAL
 public:
+    static constexpr int symlink_recursion_limit = 8; // FIXME: increase?
```

Comment:
A link to the appropriate POSIX specification would probably be nice. Also, if we now abide by the specification, the FIXME is probably obsolete.

Reply:
Removed the fixme, added a link to POSIX docs